rllib/core/learner/learner.py

```diff
@@ -286,6 +316,15 @@ def __init__(
         # name includes the ModuleID as a prefix: optimizer_name=`[ModuleID]_[.. rest]`.
         self._module_optimizers: Dict[ModuleID, List[str]] = defaultdict(list)
 
+        self._user_configured_optimizers = is_overridden(
```

Comment:
see above:
Users either:
- use RLlib's managed 1-optim per module setup.
- override `configure_optimizers_per_module`, but then have to do everything themselves (e.g. lr-scheduling, etc..).